packaging/debian/rules

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/make -f
 
-export PATH := /tmp/go/bin:$(PATH)
+export PATH := /tmp/go/bin:/usr/share/gocode/bin:$(PATH)
```

Comment:
is this for you to use the protobuf pre loaded in the build image?

Reply:
Yea, so we can access `protoc-gen-go` and `protoc-gen-go-grpc`. Otherwise we see errors that say that those aren't executable or aren't in `PATH`.